### PR TITLE
Waggle Dance Plugins: Allow jars to be added to the classpath without mvn dependency

### PR DESCRIPTION
**Pull request by courtsVII**
…aven dependency

### CODE REVIEW

**Comment by courtsVII** [comment]
resolves #93

File: waggle-dance/src/main/service/waggle-dance.conf [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #Spring boot service conf: see http://docs.spring.io/spring-boot/docs/current/reference/html/deployment-install.html#deployment-script-customization-conf-file
 LOG_FOLDER=/var/log/waggle-dance/
-JAVA_OPTS="-Xmx12g -XX:+UseG1GC -XX:G1ReservePercent=15 -Dlog4j.configurationFile=/opt/waggle-dance/conf/log4j2.xml -Dlogging.config=/opt/waggle-dance/conf/log4j2.xml"
+JAVA_OPTS="-Xmx12g -XX:+UseG1GC -XX:G1ReservePercent=15 -Dloader.path=lib,/opt/waggle-dance/lib/ -Dlog4j.configurationFile=/opt/waggle-dance/conf/log4j2.xml -Dlogging.config=/opt/waggle-dance/conf/log4j2.xml"
```
Comment (ddcprg) [inline waggle-dance/src/main/service/waggle-dance.conf]: does this not work with the jar layout?

Comment (courtsVII) [reply waggle-dance/src/main/service/waggle-dance.conf]: Not with how we have it prior to this PR

File: waggle-dance-core/pom.xml [hunk 2]
```diff
@@ -287,6 +287,7 @@
         <artifactId>spring-boot-maven-plugin</artifactId>
         <version>${spring-boot-maven-plugin.version}</version>
         <configuration>
+          <layout>ZIP</layout>
```
Comment (massdosage) [inline waggle-dance-core/pom.xml]: Can you provide an explanation of what exactly this achieves? Perhaps you could unzip the .tgz that we produce with and without this change and run `tree` over the files/folders so we can see? (I'd try this out myself but I don't have the time right now!)

**Comment by massdosage** [comment]
Not clear why this is needed as no other users have reported this as an issue.

**Pull request closed without merging**